python/lsst/afw/image/visitInfo.cc

```diff
@@ -47,7 +47,7 @@ namespace {
 static double const nan(std::numeric_limits<double>::quiet_NaN());
 static geom::Angle const nanAngle(nan);
 
-}  // anonymous
+}  // namespace
```

Comment:
Please consider moving the clang-formatting changes to the previous commit, since they really don't seem to be relevant to the important changes.